fast_excel.gemspec

```diff
@@ -7,7 +7,7 @@ Gem::Specification.new do |s|
   s.summary     = %q{Ultra Fast Excel Writer}
   s.description = "Wrapper for libxlsxwriter using ffi"
   s.license     = 'MIT'
-  s.required_ruby_version = '~> 2.0'
+  s.required_ruby_version = '~> 3.0'
```

Comment:
Why is this needed?!
It makes sense adding 3.0 to Travis matrix, but why do you need so break compatibility?

Reply:
Sry my fault, it should be `>= 2.0`, so we allow ruby 3 to be used but require at least 2.0.

Current `~> 2.0` will only allow 2.x but not 3.

Reply:
I don't think anyone still use ruby 1.9 now, but if there will be ruby 4.0 or ruby 5.0 with breaking changes I don't want bundler to choose older version because because we have `>= 2.0`. And probably we can't set it to `['>= 2.0', '< 4.0']`

So probably `< 4.0` will be more safe

Reply:
According to https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Gemspec/RequiredRubyVersion it can be set to 

```
spec.required_ruby_version = ['>= 2..0', '< 4.0']
```

Will change that.